pkg/trustedresources/verify.go

```diff
@@ -157,13 +211,47 @@ func prepareObjectMeta(in metav1.ObjectMeta) (metav1.ObjectMeta, []byte, error)
 	return out, signature, nil
 }
 
-// getVerifiers get all verifiers from configmap
-func getVerifiers(ctx context.Context, k8s kubernetes.Interface) ([]signature.Verifier, error) {
-	cfg := config.FromContextOrDefaults(ctx)
+// getVerifiersFromPolicy get all verifiers from configmap
+func getVerifiersFromPolicy(ctx context.Context, k8s kubernetes.Interface, source string, verificationpolicy *v1alpha1.VerificationPolicy) ([]signature.Verifier, error) {
 	verifiers := []signature.Verifier{}
-	// TODO(#5527): consider using k8s://namespace/name instead of mounting files.
+	for _, a := range verificationpolicy.Spec.Authorities {
+		algorithm, err := hashAlgorithm(a.Key.HashAlgorithm)
+		if err != nil {
+			continue
+		}
+		if len(a.Key.Data) != 0 {
+			for _, d := range a.Key.Data {
```

Comment:
Should we be handling KMS also here?

Reply:
KMS is not supported in this PR, I plan to pull it in next PR.